data/wanderer/wanderers start.txt

```diff
@@ -179,67 +183,388 @@ mission "Wanderers: Hai Diplomat"
+
+	`As you are approaching Hai-home, you contact the Hai government and inform them that you have found new aliens to the north. You submit a request for ambassadorial assistance to help you communicate with the aliens to the north, who seem to speak the Hai language. They immediately direct you to a now familiar private government hangar, where a familiar diplomat meets you.`
+		to display
+			not "Wanderers: Diplomacy Delayed: offered"
+	`As you are approaching Hai-home, you contact the Hai government and inform them you are ready to receive an ambassador to the aliens to the north. They immediately direct you to a now familiar private government hangar, where a familiar diplomat meets you.`
+		to display
+			has "Wanderers: Diplomacy Delayed: offered"
+	`	"Good to see you again Captain <last>, I have arranged for some refreshments so please, come and tell me all about these new aliens you have discovered!"`
+	`	Her exuberance is catching and after recent stressful events you see a new side to her as you relay your experience with the bird-like aliens. You realize that opportunities like this are why she became a diplomat in the first place and that this is, to some degree, a dream come true.`
+	`	"So in conclusion, it seems you need a translator," she says with a wry smile.`
+	choice
+		`	"Indeed, I'm hoping you have someone in mind."`
+		`	"It would be great to work with you again if you're willing."`
+	`	You open your mouth to respond but the effort is pointless.`
+	`	"I accept!" she cries, leaning forward on the desk with a gleeful look in her eyes.`
+	`	"A first contact is a dream opportunity, and there are so very few of them. So of course I will come with you."`
+	choice
+		`	"Let's go visit these aliens immediately!"`
+			goto "let's go"
+		`	"Can a distinguished ambassador such as yourself just leave like that?"`
+		`	"What if the humans want our own envoy to these aliens?"`
+			goto envoy
+	`	"Yes," she says. "If these aliens will provide a place for me, I mean to live with them permanently, an ambassador between their people and ours. I am old, and in need of a great novelty like this to make my life interesting once more." You note that the fur on her face, especially on the snout, has turned from brown to silvery gray - a sign of age in terrestrial creatures, and perhaps in the Hai as well.`
+	# Sayari is the head of the entire Hai diplomatic corps, so she doesn't need permission to choose herself as the diplomat.
+	# Going to diplomatic hot-spots is literally her job.
+	choice
+		`	"Let's go visit these aliens immediately!"`
+			goto "let's go"
+		`	"What if the humans want our own envoy to these aliens?"`
+	label envoy
+	`	She says, "If they prove to be friendly, I will help you to find a way to communicate with them directly, so you need not use me as an intermediary. But our first step must be to speak with them." You help her aboard your ship, and get ready to return to the alien homeworld.`
+		goto "let's go"
```

Comment:
```suggestion
		accept
```
`You help her aboard your ship, and get ready to return to the alien homeworld. “Excellent, I will await you in your ship.”` doesn’t really work.

Reply:
I'll test this later.

Reply:
I've fixed that dialog. I couldn't change the goto to an `accept` because that would skip the important `action` section. The changes were slightly more complicated, but they achieve what you suggested.